### PR TITLE
lvm: fix logical error

### PR DESCRIPTION
**Pull request by pingl2016**
1. Correct function logic
2. Use polling to remove logical volume device
3. When create a new physical volume device and Volume Group, bind them in setup_vg()

Signed-off-by: Ping Li pingl@redhat.com

ID: 1375876

### CODE REVIEW

**Comment by PandaWei** [comment]
LGTM

File: virttest/lvm.py [hunk 1]
```diff
@@ -294,16 +296,23 @@ def create(self):
...
-        self.umount()
-        cmd = "lvm lvremove %s %s/%s" % (extra_args, self.vg.name, self.name)
-        process.system(cmd)
-        logging.info("logical volume(%s) removed", self.name)
+        :param timeout: timeout in seconds;
+        """
+        end_time = time.time() + timeout
+        while time.time() < end_time:
+            time.sleep(0.5)
+            self.umount()
```
Comment (will-Do) [inline virttest/lvm.py]: Logically, sleep should after umount and the following step

Comment (pingl2016) [inline virttest/lvm.py]: Basically there's not much difference

**Comment by CongLi** [comment]
LGTM,
acked-by: Cong Li <coli@redhat.com>

**Pull request merged**